functions/APIs/farms.js

```diff
@@ -0,0 +1,97 @@
+const { db } = require('../util/admin');
+
+exports.getAllFarms = (request, response) => {
```

Comment:
Some documentation of these endpoints would be nice.  This one is pretty obvious given that it takes no parameters, but in other cases it's important to know what request parameters are expected.